xarray/plot/facetgrid.py

```diff
@@ -110,7 +111,8 @@ def __init__(
         size : scalar, optional
             Height (in inches) of each facet. See also: ``aspect``
         subplot_kws : dict, optional
-            Dictionary of keyword arguments for matplotlib subplots
+            Dictionary of keyword arguments for Matplotlib subplots
+            (:func:`matplotlib.pyplot.subplots`).
```

Comment:
I would use `:py:func:` here:
```suggestion
            (:py:func:`matplotlib.pyplot.subplots`).
```

Reply:
Thanks @keewis, I can make that change in #5285 as well.